New package: Effort v0.4.1

JuliaRegistrator opened the pull request:
- Registering package: Effort
- Repository: https://github.com/CosmologicalEmulators/Effort.jl
- Created by: @marcobonici
- Version: v0.4.1
- Commit: eae8f49bf9b88e890905ccf3e87c7696201ac831
- Reviewed by: @marcobonici
- Reference: https://github.com/CosmologicalEmulators/Effort.jl/commit/eae8f49bf9b88e890905ccf3e87c7696201ac831#commitcomment-168985026
- Description: Repository containing the EFfective Field theORy surrogaTe
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd648198f098909e1dbd774972ebe4720b92e731654fda89670bb1c7f4c2318e1370de1235b95db1aa51390d9a49c1c04bafa7b39c56ad9d9e393ce08294da73af5942aca97d923684165db5f1a49f9bb910bdecb558684c2d05c078b43e5fd96c4e04b91b17ba276dc5cef7269059d302b127c0a3f028cd32096ee0e3080a0f45e876fae954f0f535be3380f2b141ff83e2c615470d4bfdbb793068922b550bef7b6b5c85e25568cb0c95793cd930ab5b2 -->

marcobonici commented:
Hi @goerz , I have fixed the issue with the compat and, as suggested by the automated message, I have retriggered JuliaREgistrator (see [here](https://github.com/CosmologicalEmulators/Effort.jl/commit/bd8e4f0102871d2327fa605dccf6c822e80d8d7f) ). However, I get the error message `Error while trying to register: "Tag with name v0.4.1 already exists and points to a different commit"` . What should I do? Thanks! [noblock]

goerz commented:
Good question. It looks to me like you're doing the right thing. I'm assuming you deleted the tag more than three hours ago. Maybe not sufficiently long before? Just try commenting on https://github.com/CosmologicalEmulators/Effort.jl/commit/eae8f49bf9b88e890905ccf3e87c7696201ac831 again.

Otherwise, this seems like something to ask about on Slack.

[noblock]

marcobonici commented:
I will try again tomorrow (to let some time pass) and then I will eventually ask on Slack. The only other thing I can think of is deleting from gh-pages branch the corresponding folder, but I am not sure how this could interfer with the process...

JuliaRegistrator committed:
New package: Effort v0.4.1

UUID: ace19e24-02c2-4bce-bc41-b57e89c0fc53
Repo: https://github.com/CosmologicalEmulators/Effort.jl.git
Tree: f7ddf83a9747b00f1aa6e17ca6eeba225b858a95

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
